acs-integration-tests/src/test/java/com/ge/predix/integration/test/PolicyEvalCachingWithGraphDBIT.java

```diff
@@ -0,0 +1,272 @@
+package com.ge.predix.integration.test;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.HashSet;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.core.env.Environment;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.oauth2.client.OAuth2RestTemplate;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.testng.AbstractTestNGSpringContextTests;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import com.fasterxml.jackson.core.JsonParseException;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.ge.predix.acs.model.Attribute;
+import com.ge.predix.acs.model.Effect;
+import com.ge.predix.acs.rest.BaseResource;
+import com.ge.predix.acs.rest.BaseSubject;
+import com.ge.predix.acs.rest.Parent;
+import com.ge.predix.acs.rest.PolicyEvaluationRequestV1;
+import com.ge.predix.acs.rest.PolicyEvaluationResult;
+import com.ge.predix.test.TestConfig;
+import com.ge.predix.test.utils.ACSRestTemplateFactory;
+import com.ge.predix.test.utils.PolicyHelper;
+import com.ge.predix.test.utils.PrivilegeHelper;
+import com.ge.predix.test.utils.UaaTestUtil;
+import com.ge.predix.test.utils.ZacTestUtil;
+import com.ge.predix.test.utils.ZoneHelper;
+
+@ContextConfiguration("classpath:integration-test-spring-context.xml")
+@SuppressWarnings({ "nls" })
+public class PolicyEvalCachingWithGraphDBIT extends AbstractTestNGSpringContextTests {
+
+    @Value("${acsUrl:http://localhost:8181}")
```

Comment:
Does this setting come from the environment? If so, shouldn't we name it as per environment variable conventions i.e. ACS_URL?